docs/cedarling/integrations/cedarling-opa.md

```
@@ -0,0 +1,317 @@
+---
+tags:
+  - Cedar
+  - Cedarling
+  - OPA
+---
+
+# Cedarling OPA Plugin
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)